NOTE-act-rules-common-aspects.bs

```diff
@@ -9,16 +9,20 @@ TR: https://www.w3.org/TR/
 Group: act-framework
 Editor: Wilco Fiers, Deque Systems
 Editor: Maureen Kraft, IBM Corp.
-Abstract: Working Group note to identify common aspects under test.
+Abstract: This document is a companion to the [Accessibility Conformance Testing (ACT) Rules Format 1.0](https://w3c.github.io/wcag-act/act-framework.html) specification. It lists Common Aspects Under Test as defined by the ACT Rules Format 1.0 specification. This document is informative. It provides a reference to well defined aspects to assist authors in writing ACT Rules and to support the consistency of ACT Rules. 
 Markup Shorthands: markdown yes
 </pre>
 
 Introduction {#intro}
 =====================
 
-<p class="note" role="note">
-"Editor's note: **DRAFT** This note captures Common Aspects Under Test. The TR metadata needs to be updated when ACT rules has a TR url."
-</p>
+The term [Aspects Under Test](https://w3c.github.io/wcag-act/act-rules-format.html#input-aspects) is defined by the [Accessibility Conformance Testing (ACT) Rules Format 1.0 Specification](https://w3c.github.io/wcag-act/act-framework.html). An aspect is a distinct part of a [test subject](https://w3c.github.io/wcag-act/act-rules-format.html#test-subject). Atomic rules are required to list aspects under test in the [applicability](https://w3c.github.io/wcag-act/act-rules-format.html#applicability) and [expectations](https://w3c.github.io/wcag-act/act-rules-format.html#expectations). 
```

Comment:
Again, "Aspects Under Test" should be "Input Aspects".

Reply:
https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/w3c/wcag-act/b7ea4f761206c8a5771f1a5b3ade8d18f5e47f85/NOTE-act-rules-common-aspects.bs